src/lib.rs

```diff
@@ -1095,7 +1100,7 @@ mod tests {
     #[test]
     fn send_constraints() {
         #[derive(Default)]
-        struct LocalActor(Rc<()>);
+        struct LocalActor;
```

Comment:
Ah sorry, this should actually be:
```suggestion
        struct LocalActor {
            _ensure_not_send_not_sync: Rc<()>,
        }
```
...the intention is that this tests tonari_actor still provides an API to spawn actors that are neither Send nor Sync.

I had this in a local `fix-clippy` branch for a week already, but forgot to push/open PR. :shrug: